src/main/java/com/openlattice/edm/EdmApi.java

```diff
@@ -367,6 +373,13 @@ Void forceRemovePropertyTypeFromEntityType(
     @GET( ENTITY_SETS_BASE_PATH )
     Iterable<EntitySet> getEntitySets();
 
+    @GET( SUMMARY_BASE_PATH )
+    Map<UUID, Stream<PropertySummary>> getAllPropertySummaries( @Body EdmRequest request );
```

Comment:
Also don't use Stream here either. Iterable or a collection is acceptable as the total size for this request is expected to not be too large.